src/prepack-cli.js

```diff
@@ -241,8 +247,20 @@ function run(
           reactOutput = (arg: any);
           reproArguments.push("--reactOutput", reactOutput);
           break;
-        case "repro":
+        case "reproOnFatal":
+          reproOnFatal = true;
```

Comment:
Lot's of code duplication between "reproOnFatal" and "reproUnconditionally". Rather have a single "reproMode".